sdk/src/wallet/storage/constants.rs

```diff
@@ -4,15 +4,26 @@
 /// The default storage path.
 pub const DEFAULT_STORAGE_PATH: &str = "./storage";
 
+/// The default jammDB storage path.
+#[cfg(feature = "jammdb")]
+pub const JAMMDB_FOLDERNAME: &str = "walletdb";
+
 /// The default RocksDB storage path.
 #[cfg(feature = "rocksdb")]
 pub(crate) const ROCKSDB_FOLDERNAME: &str = "walletdb";
 
 pub const fn default_storage_path() -> &'static str {
-    #[cfg(feature = "rocksdb")]
-    return ROCKSDB_FOLDERNAME;
-    #[cfg(not(feature = "rocksdb"))]
-    DEFAULT_STORAGE_PATH
+    cfg_if::cfg_if!{
+        if #[cfg(feature = "rocksdb")]{
+            return ROCKSDB_FOLDERNAME;
```

Comment:
Do these need to be returns?

Reply:
They were returns before. I just added cfg_if enclosures to make it look pretty.

Reply:
Yes, but they don't need to be anymore.

Reply:
removed them now